README.md

```diff
@@ -19,7 +19,7 @@ to a large cluster of cloud instances.
 
 A cluster of Hazelcast nodes share both the data storage and computational load
 which can dynamically scale up and down. When you add new nodes to the cluster,
-the data is automatically rebalanced across the cluster and currently running
+the data is automatically rebalanced across the cluster. Currently running
```

Comment:
>Split two thoughts into seperate sentences instead of joining them with "and"
I won't say those were two thoughts, rather two things that are happening in this _when_ "section".